models/Petition.model.js

```diff
@@ -3,10 +3,29 @@ const Schema = mongoose.Schema;
 
 const petitionSchema = Schema(
   {
+    userId: Number,
+    participants: { type: Schema.ObjectId, ref: "Participant" },
+    loanAmount: Number,
+    status: ["pending", "inProgress", "completed"],
+    actionType: ["receive", "provide", "deliver", "borrow"],
+    startCoords: {
```

Comment:
Can we change this to be `startLoc` because it includes information other than coords as well?